httpretty/__init__.py

```diff
@@ -25,32 +25,35 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 from __future__ import unicode_literals
 
-__version__ = version = '0.8.5'
+__version__ = version = '0.8.8'
 
-from .core import httpretty, httprettified
-from .errors import HTTPrettyError, UnmockedError
-from .core import URIInfo
-
-HTTPretty = httpretty
-activate = httprettified
-
-enable = httpretty.enable
-register_uri = httpretty.register_uri
-disable = httpretty.disable
-is_enabled = httpretty.is_enabled
-reset = httpretty.reset
-Response = httpretty.Response
-
-GET = httpretty.GET
-PUT = httpretty.PUT
-POST = httpretty.POST
-DELETE = httpretty.DELETE
-HEAD = httpretty.HEAD
-PATCH = httpretty.PATCH
-OPTIONS = httpretty.OPTIONS
-CONNECT = httpretty.CONNECT
-
-
-def last_request():
-    """returns the last request"""
-    return httpretty.last_request
+try:
+    from .core import httpretty, httprettified
+    from .errors import HTTPrettyError, UnmockedError
+    from .core import URIInfo
+    
+    HTTPretty = httpretty
+    activate = httprettified
+    
+    enable = httpretty.enable
+    register_uri = httpretty.register_uri
+    disable = httpretty.disable
+    is_enabled = httpretty.is_enabled
+    reset = httpretty.reset
+    Response = httpretty.Response
+    
+    GET = httpretty.GET
+    PUT = httpretty.PUT
+    POST = httpretty.POST
+    DELETE = httpretty.DELETE
+    HEAD = httpretty.HEAD
+    PATCH = httpretty.PATCH
+    OPTIONS = httpretty.OPTIONS
+    CONNECT = httpretty.CONNECT
+    
+    
+    def last_request():
+        """returns the last request"""
+        return httpretty.last_request
+except ImportError:
+    pass
```

Comment:
I'm not sure if this `try/except` is a good thing. That will cause a `NameError` if the imports fail. A misleading exception, I would say